### DISCUSSION TITLE
rename wp login

### DISCUSSION THREAD

**Discussion by Alyia** [community/community#48786] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

@ellatrix @lkraav Hi, using the rename-login with pleasure, but WP removed it from plusgin store stating there is a security leak.
https://nl.wordpress.org/plugins/rename-wp-login/
Is it possible to update this until now reliable plugin?

Thx for answering,
Alyia

**Comment by lkraav** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> @ellatrix @lkraav Hi, using the rename-login with pleasure, but WP removed it from plusgin store stating there is a security leak.
> https://nl.wordpress.org/plugins/rename-wp-login/
> Is it possible to update this until now reliable plugin?

I'm not the author, @ellatrix is. I'm in a similar situation that there's no good replacement that I know of for my apps.

I've been thinking about forking, and just removing the admin screen portion that I think has security risks. In that sense it'd become a developer-oriented solution.

I'm still evaluating what options are available.
